prow/config.yaml

```diff
@@ -11244,7 +11244,7 @@ periodics:
           cpu: 6
           memory: "16Gi"
 
-- interval: 1h
+- cron: '0 17 24 4 *' # Run at 17:00PST (2:00 CEST) on 24th April
```

Comment:
I think the timezone used by cron is UTC (which is 2 hours behind us). See for e.g gke-large-performance job above.

Reply:
Good point. Fixed.